src/ray/core_worker/test/mock_worker.cc

```diff
@@ -37,6 +39,19 @@ class MockWorker {
     // Note that this doesn't include dummy object id.
     RAY_CHECK(return_ids.size() >= 0);
 
+    if (args.size() == 1 && args[0]->GetData()->Size() == c_get_pid_string.size() &&
+        memcmp(args[0]->GetData()->Data(), c_get_pid_string.data(),
+               c_get_pid_string.size()) == 0) {
```

Comment:
This is a little bit too ad-hoc.  Better use function descriptors to distinguish the purposes of tasks, and define 2 different functions to handle the tasks.

Reply:
Thanks.I have define 2 functions to handle the tasks.